windows_containers/windows-containers-release-notes-3-x.adoc

```diff
@@ -0,0 +1,103 @@
+[id="windows-containers-release-notes-3-x"]
+= Windows Container Support for Red Hat OpenShift release notes
+include::modules/common-attributes.adoc[]
+:context: windows-containers-release-notes
+
+toc::[]
+
+[id="about-windows-containers"]
+== About Windows Container Support for Red Hat OpenShift
+
+Windows Container Support for Red Hat OpenShift is a feature providing the ability to run Windows compute nodes in an {product-title} cluster. This is possible by using the Red Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. With Windows nodes available, you can run Windows container workloads in {product-title}.
+
+The release notes for Red Hat OpenShift for Windows Containers tracks the development of the WMCO, which provides all Windows container workload capabilities in {product-title}.
+
+[id="getting-support"]
+== Getting support
+
+You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
+
+[id="wmco-3-0-0"]
+== Release notes for Red Hat Windows Machine Config Operator 3.0.0
+
+This release of the WMCO provides bug fixes and enhancements for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 3.0.0 were released in link:https://access.redhat.com/errata/RHBA-2021:xxxx[RHBA-2021:xxxx].
+
+// Replace above advisory link when known (available from Errata tool once live ID is assigned)
+
+WMCO supports self-managed clusters built using installer-provisioned infrastructure running on the following cloud providers:
+
+// The content below may require updates based on BYOH epic.
+
+* Amazon Web Services (AWS)
+* Microsoft Azure
+* VMware vSphere
+
+The following Windows Server operating systems are supported in this release of the WMCO, depending on which platform your cluster is installed on:
+
+[cols="1,4",options="header"]
+|===
+
+|WMCO platform
+|Windows Server version
+
+|AWS
+|Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019
+
+|Azure
+|Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019
+
+|vSphere
+|Windows Server Semi-Annual Channel (SAC): Windows Server 2004 with link:https://support.microsoft.com/en-us/topic/august-11-2020-kb4566782-os-build-19041-450-ec5cf4b6-86ed-b0d3-539c-5fbe1b17f56a[Microsoft patch KB4566782]
```

Comment:
Windows Server Semi-Annual Channel (SAC): Windows Server 2004  includes the patch already.

Reply:
So, does that mean that I don't need to note this?

Reply:
Probably, let's run it by the Team first. 

Pointed that out, because should be explicit that the manual installation of the patch KB4566782 is not required with Windows Server Semi-Annual Channel (SAC): Windows Server 2004.

Reply:
This needs to be included. Someone could potentially bring up a 2004 instance without this patch especially when creating a golden image on vSphere.